pkg/azure/aks/aks.go

```diff
@@ -113,15 +107,25 @@ func New(ctx context.Context, cfg *Config) (*Collector, error) {
 		return nil, err
 	}
 
+	populationCron := cron.New()
+	_, err = populationCron.AddFunc(fmt.Sprintf("@every %s", priceRefreshInterval), priceStore.PopulatePriceStore)
+	if err != nil {
+		return nil, err
+	}
+	_, err = populationCron.AddFunc(fmt.Sprintf("@every %s", machineRefreshInterval), machineStore.PopulateMachineStore)
+	if err != nil {
+		return nil, err
+	}
+
+	populationCron.Start()
```

Comment:
This isn't a _hard_ no, but I don't like the idea of adding a library for something that we can effectively handle with a ticker. See [go by example](https://gobyexample.com/tickers), I don't think it'd be that hard to emulate that with just tickers.

Reply:
updated to just use tickers